manual/mapper/daos/delete/README.md

```diff
@@ -105,6 +105,15 @@ The method can return:
     ResultSet deleteIfDescriptionMatches(UUID productId, String expectedDescription);
     // if the condition fails, the result set will contain columns '[applied]' and 'description'
     ```
+  
+* a [BoundStatement]. This is intended for queries with custom IF clauses; where you will execute
+  this statement later or in a batch.
+  
+    ```java
+    @Delete(entityClass = Product.class, customIfClause = "description = :expectedDescription")
+    BoundStatement deleteIfDescriptionMatches(UUID productId, String expectedDescription);
+    // if the condition fails, the result set will contain columns '[applied]' and 'description'
+    ```
```

Comment:
This should also be added to the [annotation javadocs](https://docs.datastax.com/en/drivers/java/4.3/com/datastax/oss/driver/api/mapper/annotations/Delete.html).

There's a lot of overlap between the javadocs and the manual, we try to get the information out any way we can :)

Reply:
Added the docs to the annotation classes too.